.github/workflows/unit.yml

```diff
@@ -23,9 +23,13 @@ jobs:
         uses: dapr/setup-dapr@v1
         with:
           version: '1.8.0'
-      - name: Install Docker for macOS
+      - name: Install Docker & install skywalking oapServer
         uses: docker-practice/actions-setup-docker@master
-        if: ${{ matrix.platform == 'macos-latest' }}
```

Comment:
You should not remove these two lines which is an important step to set up docker for specific platforms. Just adding another standalone step to deal with skywalking deps is the right thing to do here.